src/cd.rs

```diff
@@ -204,15 +199,14 @@ pub(crate) fn coherence_ratio(
     include_languages: Option<Vec<&'static Language>>,
 ) -> Result<CoherenceMatches, String> {
     let threshold = f32::from(threshold.unwrap_or(OrderedFloat(0.1)));
-    let mut include_languages = include_languages.unwrap_or_default();
-    let ignore_non_latin =
-        include_languages.len() == 1 && include_languages.first() == Some(&&Language::Unknown);
+    let mut include_languages: Vec<&Language> = include_languages.unwrap_or_default();
+    let ignore_non_latin = include_languages == vec![&Language::Unknown];
     if ignore_non_latin {
         include_languages.clear();
     }
 
     let mut results: CoherenceMatches = vec![];
-    let mut sufficient_match_count: u64 = 0;
+    let mut sufficient_match_count: u8 = 0;
```

Comment:
oops sorry I was mistaken. I thought it was inside the inner loop.
Int that case it would be practically impossible for it to hit 255.
but since it is reliant on the input it definitely can ( but still unlikely for most inputs) go above 255.

I will revert this change.